7.2.0/postgresql/org_data_fetch.sh

```
@@ -0,0 +1,77 @@
+
+# Check if the export file contains more than just the header
+if [[ $(wc -l < "$EXPORT_FILE") -eq 0 ]]; then
+  echo "Query executed successfully but no data returned. Exiting with success."
+  exit 0
+else
+  echo "Data exported to $EXPORT_FILE successfully in CSV format."
+fi
+
+# Unset PGPASSWORD to avoid leaving it in the environment.
+unset PGPASSWORD
+
```

Shellcheck (0.11.0)

[error]  Fix any mentioned problems and try again.

(SC1072)